lci-odoo10/odoo/addons/lci_ecole/models/cours.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 lorenzo6ioti
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class Cours(models.Model):
+    _name="ecole.cours"
+    _inherit = 'ecole.cours'
+    lieu=fields.Char(string="Lieu")
```

Comment:
pep8 syntax lieu = fields
when the name is the same than the string then it is not required to set a string attribute
warning! field name must be named in english as all the source code. Only translation file are authorized into other langage